pythran/pythonic/include/types/list.hpp

```diff
@@ -329,6 +331,12 @@ namespace pythonic
   struct assignable<types::list<T>> {
     typedef types::list<typename assignable<T>::type> type;
   };
+
+  // to cope with std::vector<bool> specialization
+  template <>
+  struct returnable<types::list<bool>::reference> {
```

Comment:
> > +  // to cope with std::vector<bool> specialization
> > +  template <>
> > +  struct returnable<types::list<bool>::reference> {
> 
> I don't get it. Why a list here?

because types::list<bool> uses std::vector<bool> for its storage and
std::vector<bool> is specialized in a way that does not interacts well
with pythran